src/mcts/search.cc

```diff
@@ -192,11 +192,17 @@ int64_t Search::GetTimeToDeadline() const {
 }
 
 namespace {
-inline float GetFpu(const SearchParams& params, Node* node, bool is_root_node) {
+inline float GetFpu(const SearchParams& params, Node* node,
+                    bool is_root_node, bool logit_q = false) {
```

Comment:
``logit_q`` doesn't need to be given here as one can already access it by ``params.GetLogitQ()``

Reply:
True, but do I always want to call GetFpu with `logit_q = params.GetLogitQ()`?

Reply:
That is indeed a good question. Currently there are 2 calls to ``GetFpu()`` without the ``logit_q`` parameter. In my opinion they should both be called with ``logit_q`` included, as one of them is ``verbosemovestats`` and the other for ``TempValueCutoff`` which we definitely would like to do in logit space anyway -- there still are missing some ``logit_q`` calls anyway.

Reply:
OK. I've switched to using the parameter rather than passing `logit_q` as an argument to the function.